gengokit/gengokit_test.go

```diff
@@ -69,17 +69,13 @@ func TestNewTemplateExecutor(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	const goImportPath = "github.com/TuneLab/go-truss/gengokit"
+	const goImportPath = "github.com/TuneLab/go-truss/gengokit/general-service"
 
-	te, err := newTemplateExecutor(dt, goImportPath)
+	te, err := newTemplateExecutor(dt, goImportPath, goImportPath)
```

Comment:
A clearer interface here, I think, would be passing `""` as the third parameter if it's going to be the same. If it's just for testing, though, then it's not a big deal.